host-configs/llnl/toss_4_x86_64_ib/clang@14.0.6.cmake

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level LICENSE file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# Example host-config file for a cluster on a toss4 platform (e.g. quartz) at LLNL
+#------------------------------------------------------------------------------
+#
+# This file provides CMake with paths / details for:
+#  C,C++, & Fortran compilers + MPI
+#------------------------------------------------------------------------------
+# clang-14.0.6 compilers
+# Uses gcc@10.3.1 standard library
+
+set(CLANG_VERSION "clang-14.0.6")
+set(CLANG_HOME "/usr/tce/packages/clang/${CLANG_VERSION}")
+set(GNU_HOME "/usr/tce/packages/gcc/gcc-10.3.1")
+
+set(CMAKE_C_FLAGS "--gcc-toolchain=${GNU_HOME}" CACHE STRING "")
```

Comment:
Do we know if this is still necessary on this machine? What is the default standard library?

Reply:
i never thought to try this. seemed to have worked locally

Reply:
```
[white238@quartz1538:~]$ which g++
/usr/tce/bin/g++

[white238@quartz1538:~]$ g++ --version
g++ (GCC) 10.3.1 20210422 (Red Hat 10.3.1-1)
```